std/array.d

```diff
@@ -2652,8 +2661,18 @@ if (isDynamicArray!(E[]) &&
 }
 
 /++
-    Returns a new array that is $(D s) with $(D slice) replaced by
-    $(D replacement[]).
+    Creates a new array such that the items in `slice` are replaced with the
+    items in `replacement`. `slice` and `replacement` do not need to be the
+    same length. The result will grow or shrink based on the items given.
+
+    Params:
+        s = the base of the new array
```

Comment:
I think 'base' is confusing, maybe use `= the original array` or 'source array'.